src/solvers/strings/format_specifier.cpp

```diff
@@ -27,8 +27,10 @@ static format_specifiert format_specifier_of_match(std::smatch &m)
   std::string flag = m[2].str();
   int width = m[3].str().empty() ? -1 : std::stoi(m[3].str());
   int precision = m[4].str().empty() ? -1 : std::stoi(m[4].str());
+  // `tT` values:  "t" = date/time, "T" = uppercase date/time
   std::string tT = m[5].str();
 
+  // date/time
```

Comment:
maybe renaming the boolean to `date_time` would be clearer than commenting on it

Reply:
I thought you wanted to keep it as the JDK implementation?